Strongbox/Classes/Strongbox.swift

```diff
@@ -155,3 +155,52 @@ public class Strongbox {
     }
 
 }
+
+// Codable
+
+extension Strongbox {
```

Comment:
Wouldn't you want to have `extension Strongbox: Codable` here?

Reply:
if I was to make Strongbox class Codable it would only make it adhere to the protocol instead of storing a Codable (Encodable/Decodable) type to the keychain. So we have specialized methods to handle converting those types to what Strongbox does already.

Reply:
Oh! Of course. Doh! Long day here. :-)